core/shortcut_items.ts

```diff
@@ -460,6 +465,166 @@ export function registerReadBlockParentSummary() {
   ShortcutRegistry.registry.register(readBlockParentSummaryShortcut);
 }
 
+/**
+ * Registers a keyboard shortcut that sets the focus to the block
+ * that owns the current focused node.
+ */
+export function registerJumpBlockStart() {
+  const jumpBlockStartShortcut: KeyboardShortcut = {
+    name: names.JUMP_BLOCK_START,
+    preconditionFn: (workspace) => {
+      return !workspace.isFlyout && focusedNodeHasBlockParent(workspace);
+    },
+    callback(workspace) {
+      const selectedBlock = workspace.getCursor().getSourceBlock();
+      if (!selectedBlock) return false;
+      getFocusManager().focusNode(selectedBlock);
+      return true;
+    },
+    keyCodes: [KeyCodes.HOME],
+  };
+  ShortcutRegistry.registry.register(jumpBlockStartShortcut);
+}
+
+/**
+ * Registers a keyboard shortcut that sets the focus to the
+ * last input of the block that owns the current focused node.
+ */
+export function registerJumpBlockEnd() {
+  const jumpBlockEndShortcut: KeyboardShortcut = {
+    name: names.JUMP_BLOCK_END,
+    preconditionFn: (workspace) => {
+      return !workspace.isFlyout && focusedNodeHasBlockParent(workspace);
+    },
+    callback(workspace) {
+      const selectedBlock = workspace.getCursor().getSourceBlock();
+      if (!selectedBlock) return false;
+      const inputs = selectedBlock.inputList;
+      if (!inputs.length) return false;
+      const connection = inputs[inputs.length - 1].connection;
+      if (!connection || !(connection instanceof RenderedConnection))
+        return false;
+      getFocusManager().focusNode(connection);
```

Comment:
Does this make it possible to focus a connection that can't be navigated to normally @maribethb @gonfunko?